hs_app_netCDF/receivers.py

```diff
@@ -87,6 +87,16 @@ def netcdf_pre_create_resource(sender, **kwargs):
                     box = {'coverage': {'type': 'box', 'value': res_dublin_core_meta['box']}}
                     metadata.append(box)
 
+                # add rights
+                if res_dublin_core_meta.get('rights'):
+                    raw_info = res_dublin_core_meta.get('rights')
+                    import re
```

Comment:
@gantian127 I think it is more efficient to put this "import re" at the top of the module rather than within this function for efficiency to make sure it is only imported once. See https://www.python.org/dev/peps/pep-0008/ for some guidelines on imports.